### ISSUE TITLE
MM montages uses image shift instead of stage shift

### ISSUE THREAD

**Issue by jchutchings** [eisfabian/SPACEtomo#6]
Hi Fabian,

I noticed that the MM montages in SPACE v1.1 is using image shift instead of stage shifts, even though "option 1" for stage montage parameters is used in the functions script. 

It looks like SPACE v1.0 used stage shifts, however I have since updated SerialEM for SPACE v1.1. Not sure whether this is a bug in SPACE or more generally in SerialEM - have you noticed the same?

Attaching my settings but let me know if you need anything else:
[settings_JH.txt](https://github.com/user-attachments/files/15841126/settings_JH.txt)

Best,
Josh

**Comment by eisfabian** [comment]
Hi Josh,

You are right, according to the SetMontageParams command, it should use stage shift.
Is is using pure image shift? Or does it use a combination of image and stage shift?

When you open the montage dialogue, is the selected default image shift?

What is it using when you run a small script like this:
```
OpenNewMontage 3 3 test.mrc
SetMontageParams 1
M
```

Thanks and best,
Fabian

**Comment by jchutchings** [comment]
Hi Fabian,

Thanks for the suggestions. Sorry for the delay, took me a while to get back on the microscope and test it!

I guess at the time of testing, the default montaging method was image shift instead of stage shift, so I changed it back and it seems fine now. I must have assumed the command in the functions script would overwrite this.

I tested your small script and it is purely stage shift, and SPACEtomo MMMs are also purely stage shift now. Apologies for the false alarm!

Best wishes,
Josh
